src/css/components/search.scss

```diff
@@ -0,0 +1,56 @@
+@import '../core.scss';
+
+$search-btn-width:   4.5rem;
+$search-btn-height:  2rem;
+
+@mixin search-icon {
+  background-image: url('../img/i-search.svg');
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.search-bar {
+  @include clearfix;
+  max-width: none;
+  position: relative;
+  $width: $search-btn-width;
+  margin-left: 0;
+  margin-top: 0.5rem;
+  margin-right: 0;
+  margin-bottom: 0;
+
+  [type='search'],
+  .search-field {
+    -webkit-appearance: none;
+    border-bottom-right-radius: 0;
+    border-right: none;
+    border-top-right-radius: 0;
+    float: left;
+    height: $search-btn-height;
+    margin: 0;
+    padding-bottom: 0;
+    padding-top: 0;
+    width: calc(100% - #{$width});
+
+    @include media($small-screen) {
+      width: calc(100% - #{width});
+    }
+  }
+
+  [type='submit'],
+  .search-submit {
```

Comment:
`type=submit` probably should, but `.search-submit` does not. I'll take that into account as I refactor the code for the new designs ^5